web/src/views/ClockConverterView.vue

```diff
@@ -0,0 +1,204 @@
+<template>
+	<div class="container">
+		<div
+			class="d-flex flex-column vh-min justify-content-center align-items-center">
+			<!-- <lottie-player
+				src="https://assets6.lottiefiles.com/packages/lf20_cdxs2q1x.json"
+				background="transparent"
+				speed="0.005"
+				style="width: 300px; height: 300px;"
+				loop
+				autoplay/> -->
+
+			<div class="col-lg-12 col-md-4 text-center">
+				<h3 class="mb-0">
+					24 Hour Time Converter
+				</h3>
+
+				<p class="mb-4">
+					<small>
+						Convert 24 Hour to 12 Hour
+					</small>
+				</p>
+
+				<ErrorAlert v-if="error">
+					{{ err }}
+				</ErrorAlert>
+
+				<div
+					v-if="time"
+					class="alert alert-primary alert-dismissible fade show"
+					role="alert">
+					<strong>
+						Converted Time:
+					</strong> 
+					{{ time }}.
+					<button
+						@click="convertedTime = ''"
+						type="button"
+						class="btn-close"
+						data-bs-dismiss="alert"
+						aria-label="Close"/>
+				</div>
+
+				<div class="mb-3">
+					<form v-on:submit.prevent>
+
+						<div class="row rounded-top ">
+							<div class="col mb-3">
+								<label 
+									for="t24HourInput" 
+									class="form-label">
+									24 Hour Clock
+								</label>
+
+								<input
+									name="t24HourInput"
+									type="text"
+									class="form-control mb-3"
+									placeholder="00:00"
+									v-model="t24HourInput"/>
+
+								<button
+									class="btn btn-outline-primary"
+									type="submit"
+									@click="convertTime24to12"
+									:disabled="t24HourInput === '' ? true : false">
+									Convert to 12-hour
+								</button>
+
+							</div>
+
+							<div class="col">
+								<label for="t12HourInput" class="form-label"
+									>12 Hour Clock
+								</label>
+
+								<input
+									name="t12HourInput"
+									type="text"
+									class="form-control mb-3"
+									placeholder="12:00 AM"
+									v-model="t12HourInput"/>
+
+								<button
+									class="btn btn-outline-primary"
+									type="submit"
+									@click="convertTime12to24"
+									:disabled="t12HourInput === '' ? true : false">
+									Convert to 24-hour
+								</button>
+							</div>
+
+						</div>
+					</form>
+				</div>
```

Comment:
This whole file's indention looks awkward to me. May I know what indention this file uses?

Reply:
I stopped using formatters , so let me figure it out.

Reply:
I just checked the code, and locally it looks fine. I'm not sure why it looks different here.